Functions.cs

```diff
@@ -2,86 +2,126 @@ namespace Company.Function;
 
     internal class Functions
```

Comment:
Functions.cs is not descriptive naming. using something like EmissionCalculator or EmissionConvertor